CHANGELOG.md

```diff
@@ -1,10 +1,27 @@
 # BetoCQ test suite release history
 
+## 2.3.3 (08/27/2024)
+
+## New
+* Add TDLS support check.
```

Comment:
can we elaborate on the impact of this addition instead of just stating we added a check?